cmd/site-replication.go

```diff
@@ -3058,3 +3272,995 @@ func (c *SiteReplicationSys) PeerEditReq(ctx context.Context, arg madmin.PeerInf
 	}
 	return nil
 }
+
+const siteHealTimeInterval = 1 * time.Minute
```

Comment:
Maybe this is too low for a large deployment

Reply:
Every minute is okay @vadmeste